docs/user/dashboard/lens.asciidoc

```diff
@@ -240,7 +250,7 @@ image::images/lens_referenceLine_7.16.png[Lens drag and drop focus state]
 [[filter-the-data]]
 ==== Apply filters
 
-You can use the <<semi-structured-search, KQL query bar>> to focus on a known set of data for the entire visualization, or use the filter options from the layer pane or legend.
+You can use the <<semi-structured-search, query bar>> to focus on a data set for the entire visualization, or use the filter options from the layer pane or legend.
```

Comment:
"**in** the layer pane or legend"?

Reply:
The user is using the query bar, but really it is the query that the user creates that focuses on the data set.